src/Postmark/PostmarkClient.cs

```diff
@@ -863,10 +863,11 @@ public async Task<PostmarkResponse> SendEmailWithTemplateAsync(TemplatedPostmark
             bool? trackOpens = null,
             IDictionary<string, string> headers = null,
             IDictionary<string, string> metadata = null,
+            string messageStream = null,
             params PostmarkMessageAttachment[] attachments)
```

Comment:
Due to the `Attachment` `params` at the end, we cannot add this argument at the end of the function signature, so we must break backwards compatibility. This probably requires a major version update.

I could also combine this major version update with the `BounceID` long conversion when making the release.

Alternatively, we could choose to not add this parameter here, because we have a different function:
`SendEmailWithTemplateAsync(TemplatedPostmarkMessage emailToSend)` which has a context object where you can pass the MessageStream cleanly. We could have this function be more limited and always use the Default stream.